src/ui/main.cpp

```diff
@@ -386,46 +394,51 @@ static void main_loop(void *arg) {
         ImGui::PopID();
     } else if (app->mainViewMode == "FlowGraph") {
         if (app->dashboard != nullptr) {
-            auto contentRegion = ImGui::GetContentRegionAvail();
+            // TODO: tab-bar is optional and should be eventually eliminated to optimise viewing area for data
```

Comment:
Additionally it is hard to use since the leftmost tab is often overlayed by the :hamburger: menu especially if it has a short title. This should probably be included into the hamburger menu at a sub-level.